cmd/state-svc/main.go

```diff
@@ -38,28 +38,29 @@ const (
 func main() {
 	var exitCode int
 
-	defer func() {
-		if panics.HandlePanics(recover(), debug.Stack()) {
-			exitCode = 1
-		}
-		if err := events.WaitForEvents(5*time.Second, rollbar.Wait, rollbar.Close, authentication.LegacyClose, logging.Close); err != nil {
-			logging.Warning("Failing to wait for rollbar to close")
-		}
-		os.Exit(exitCode)
-	}()
-
 	cfg, err := config.New()
 	if err != nil {
 		logging.Critical("Could not initialize config: %v", errs.JoinMessage(err))
+		os.Stderr.WriteString(locale.Tr("err_main_config", errs.JoinMessage(err)))
```

Comment:
I wouldn't use locale here, this early on in the execution that could open us up to some brand new type of bugs.

Reply:
Fair enough. Note that we are using `locale` pretty early in the state tool here: https://github.com/ActiveState/cli/blob/master/cmd/state/main.go#L76. I can sneak in a fix for this and other instances of locale so early in this PR if that's something we want.